src/Avalonia.Animation/AnimationInstance`1.cs

```diff
@@ -82,6 +79,7 @@ protected override void Unsubscribed()
             // Animation may have been stopped before it has finished.
             ApplyFinalFill();
 
+            _targetControl.PropertyChanged -= ControlPropertyChanged;
```

Comment:
Not sure how often we create this animation instance but we might want to cache this delegate too. Since right now we will allocate once to subscribe and then again to unsubscribe.

Reply:
yeah animationinstance is always disposed of for every cycle of the animation, would be great if that's cached

Reply:
Done.